src/pages/_app.tsx

```diff
@@ -60,6 +67,25 @@ export default function App(props: AppProps & { emotionCache?: EmotionCache }) {
   return (
     <CacheProvider value={emotionCache}>
       <Head>
+        <title key="title">strompreis.elcom.admin.ch/</title>
+        <meta property="og:type" content="website" />
+        <meta
+          property="og:title"
+          content={t({
+            id: "app.meta.title",
+            message: "Electricity tariffs in Switzerland",
+          })}
+        />
+        <meta
+          property="og:description"
+          content={t({
+            id: "app.meta.description",
+            message:
+              "Detailed price analyses of cantons, municipalities and grid operators.",
+          })}
+        />
+        <meta property="og:image" content={`${PUBLIC_URL}/og-image.png`} />
+        <meta property="og:url" content={`${PUBLIC_URL}${asPath}`} />
```

Comment:
I wonder how the `t` is going to work if not in I18nProvider ? I guess it works since you worked, do you know what happens if not in I18nProvider ?
I think this should go in its own component OpenGraphTags, this would be easier to find.